onnxmltools/convert/sklearn/operator_converters/NaiveBayes.py

```diff
@@ -100,7 +104,35 @@ def convert_sklearn_naive_bayes(scope, operator, container):
     #                       reshaped_result[M,]             --------------------------------------RESHAPE
     #                                   |                   |
     #                                   V                   |
-    #  onnx_proto.TensorProto.INT64 -> CAST -> output[M,] <-|
+    # (to=onnx_proto.TensorProto.INT64)CAST -> output[M,] <-|
+    #
+    #
+    # If model's binarize attribute is not null, then input is updated based on the following graph:
+    #
+    #    input[M, N] -> GREATER <- threshold[1]
+    #       |              |
+    #       |              V
+    #       |       condition[M, N] -> CAST(to=onnx_proto.TensorProto.FLOAT) -> cast_values[M, N]
+    #       |                                                                       |
+    #       V                                                                       V
+    #   CONSTANT_LIKE ---------------------------> zero_tensor[M, N] ------------> ADD
+    #                                                                               |
+    #                                                                               V
+    #                                                   input[M, N] <- binarised_input[M, N]
+    #
+    # Sub-graph for probability calculation common to both Multinomial and Bernoulli Naive Bayes
+    #
+    #  sum_result[M, C]* -> REDUCELOGSUMEXP -> reduce_log_sum_exp_resulti[M,] -|
```

Comment:
I still fail to infer the meaning of `*`...

Reply:
[M,] represents a vector. For example:
np.array([1, 2]).shape
gives shape [2,]. I am then using a reshape to convert this vector to a matrix of shape [2, 1]. 
Also, * is used to denote that sum_result was calculated previously and we are using that here. I have marked this with * in the previous graph as well to link them. Is this confusing?

Reply:
I have removed *.